src/app/modules/wallet/wallet-send/wallet-send.component.html

```diff
@@ -148,7 +148,15 @@ <h5>Interact with a Smart Contract on GoChain</h5>
                      *ngFor="let functionInput of selectedFunction.inputs; let i=index"
                      class="uk-margin">
                   <div class="uk-form-controls">
-                    <input class="uk-input" placeholder="{{functionInput.name}}" formControlName="{{i}}">
+                    <ng-container *ngIf="selectedFunction.name === 'transfer' && functionInput.name === 'value' && addr.token_symbol; else regularFunctionInput">
+                      <input type="number" class="uk-input" placeholder="Value ({{addr.token_symbol}})" formControlName="{{i}}" #valueInput>
+                      <p *ngIf="valueInput.value">
+                        Raw value: {{valueInput.value * ('1e' + addr.decimals)}}
```

Comment:
Do we need to use the big number library here?

Reply:
@jmank88  actually no, cause we miltiply

Reply:
But doesn't `1000000000000000000` overflow?

Reply:
Or worse:
`10000.000000000001`
`10000.000000000000000001`

Reply:
Quick JS playground code shows those inputs losing precision and displaying as exponents, but I'm out of my element here:
`1.0000000000000002e+22`
`1e+22`

Reply:
Added bignumber 

<img width="801" alt="Screenshot 2019-05-10 at 22 37 21" src="https://user-images.githubusercontent.com/42649300/57542694-31c67100-7374-11e9-90fc-07ef8615e999.png">

Reply:
Where was bignumber added? Do we need another separate PR now?
Can you try these numbers as _inputs_:
`10000.000000000001`
`10000.000000000000000001`

Reply:
#301